src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -155,6 +155,49 @@ TEST_CASE("CheckCopyMovingVanillaNetworkTest", "[FeedForwardNetworkTest]")
   CheckMoveFunction<>(model1, trainData, trainLabels, 1);
 }
 
+/**
+ * Special test case for PReLU layer.
+ * To be removed once PReLU is corrected.
+ */
+TEST_CASE("PReLULoadSaveTest", "[FeedForwardNetworkTest]")
```

Comment:
You can reuse:

https://github.com/mlpack/mlpack/blob/149688fc9c7472f2e02463aeafd6d29a4aa073a2/src/mlpack/tests/feedforward_network_test.cpp#L779-L826

Just alter the network structure to use the `PReLU`  instead of `SigmoidLayer`.

Reply:
Thanks
It seems to be passing in all environments.